packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,222 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, DfTxType } from '../../../src/category/account'
+import { createToken, accountToAccount } from '@defichain/testing'
+
+describe('Account', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const burnAddress = 'mfburnZSAM7Gs1hpDeNaMotJXSGA7edosG'
+  const burnAddressPrivateKey = '93ViFmLeJVgKSPxWGQHmSdT5RbeGDtGW4bsiwQM2qnQyucChMqQ'
+  let fundedAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+
+    await client.wallet.importPrivKey(burnAddressPrivateKey)
+
+    await createHistory()
+  })
+
+  const createHistory = async (): Promise<void> => {
```

Comment:
I would consider this as top level function.  See
https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#top-level-functions-should-not-use-arrow-syntax

You may write like this
`async function createHistory(): Promise<void> {  `

which is the common way written by other developer